scripts/tests/test_zkai_d64_commitment_consistency_method_probe.py

```
@@ -215,6 +219,28 @@ def test_validation_rejects_manifest_drift(self) -> None:
+        with self.assertRaisesRegex(PROBE.CommitmentConsistencyProbeError, "commitment mismatch"):
+            PROBE.validate_probe(payload)
+
+    def test_validation_rejects_malformed_manifest_without_keyerror(self) -> None:
+        payload = PROBE.build_probe()
+        del payload["proof_native_parameter_manifest"]["proof_native_parameter_commitment"]
+        payload["proof_native_parameter_manifest_commitment"] = PROBE.blake2b_commitment(
+            payload["proof_native_parameter_manifest"],
+            "ptvm:zkai:d64:proof-native-parameter-manifest-payload:v1",
+        )
+
+        with self.assertRaisesRegex(PROBE.CommitmentConsistencyProbeError, "missing commitment"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)